main.go

```diff
@@ -89,6 +91,26 @@ func mainerr() error {
 		}
 	}
 
+	*fTags = strings.TrimSpace(*fTags)
+	if gf := os.Getenv("GOFLAGS"); gf != "" {
```

Comment:
I wonder whether if the command line tags are specified that it should override the env var. What do you think?

Reply:
Here I'm simply following what `cmd/go` does; from `go help environment`:

```
GOFLAGS
        A space-separated list of -flag=value settings to apply
        to go commands by default, when the given flag is known by
        the current command. Flags listed on the command-line
        are applied after this list and therefore override it.
```